### PR TITLE
Update README.md file

### PR DESCRIPTION
**Pull request by eudondian**
### CRYPTO-COINRANKING APP

- The Project was set up, created the app, installed dependencies like redux, redux-toolkit...
- I created the file structure using the duck method
- I created a service file called cryptoApi.js to fetch API data
- I used a Filter stateless component to filter data retrieved from the API
- The fetched data was then rendered on the browser
- I dispatched an action to store the selected data in the Redux store
- Antd was used to style the UI
- Updated README.md file
- https://www.loom.com/share/a2d4378e5728467fa49aa3a1ee7e46ee

### CODE REVIEW

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @eudondian, 

Good job so far 🔥!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🎉
- Styles ✔
- Routing ✔
- Linters ✔

# Required Changes ♻️♻️
- [ ] Let's add a details page
- [ ] Let's add the presentation video
- [ ] Let's deploy the app
- [ ] Let's add a link to the deployed project in the README.
- [ ] Let's add Uses a Filter stateless component to filter data retrieved from the API

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Commit all your changes in the PR branch so that they appear in the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 1]
```diff
@@ -17,6 +18,7 @@ const App = () => (
           <Routes>
             <Route exact path="/" element={<Homepage />} />
             <Route exact path="/cryptocurrencies" element={<Cryptocurrencies />} />
+            <Route exact path="/cryptodetails" element={<CryptoDetails />} />
```
Comment (mateo951) [inline src/App.js:21]: Addressed ✔

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @eudondian, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

## ❗ Required Changes ❗
- As the previous reviewer mentioned, please be sure to add your live link in your readme file 🤓

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by eudondian** [comment]
Hello Mateo

Hope this meets you well


I have added a live link to my README file as requested by the previous reviewer

the live link has been added on GitHub (dev branch)

If I push it via the terminal, there will be a conflict

Kindly confirm this. 
Thank you

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @eudondian, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You did well to make the changes requested by the previous reviewer✔️✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**